cli/src/pcluster/templates/cw_dashboard_builder.py

```diff
@@ -39,6 +42,10 @@ def new_pcluster_metric(title=None, metrics=None, supported_vol_types=None):
     return _PclusterMetric(title, metrics, supported_vol_types)
 
 
+def custom_metric_filter(error_information=None, metric_name=None, filter_pattern=None):
+    return _CustomMetricFilter(error_information, metric_name, filter_pattern)
```

Comment:
Since this function isn't performing any logic, can we use `_CustomMetricFilter` directly?  Especially since its usages are in the same file.

Reply:
Done